### ISSUE TITLE
how to deal with CSS Images ?

### ISSUE THREAD

**Issue by andresclua** [blonestar/wp-theme-vite-tailwind-jit#7]
hey, what's the best way to deal with images on CSS/SCSS files?
It's not rendering.

![Captura de Pantalla 2022-10-19 a la(s) 19 47 29](https://user-images.githubusercontent.com/612806/196766742-c75f5eb0-6b57-4320-b8ef-9bd3a535be84.png)

**Comment by robinvoor** [comment]
I have the same question. Also, (how) can local fonts be loaded?

**Comment by andresclua** [comment]
hey @robinvoor are you working on this currently?
I found 0 information on blogs, and a lot of people has faced this issue.

**Comment by blonestar** [comment]
Hey @andresclua, fastest way is to just correct the url
`
background-image: url(wp-content/themes/wp-theme-vite-tailwind/assets/img/hello-world.jpg);
`

- or to generate url (or css variable) with php
- or to use Vite engine to move images into build (example: /dist/images) folder, I have never play with that
  but you can see the example here
  https://github.com/vitejs/vite/discussions/6552

**Comment by blonestar** [comment]
Hi @robinvoor,
fonts can be loaded in several ways:

- within css (search for "css font-face") - I suggest adding under Tailwind @layer base
![image](https://user-images.githubusercontent.com/5240133/197906890-04c73b18-53fe-4112-826b-0bb9fad534e1.png)

- WP (php) hook
![image](https://user-images.githubusercontent.com/5240133/197907344-d55308b3-11d0-42f7-8713-15bf55d41102.png)

- theme.json file (latest)
![image](https://user-images.githubusercontent.com/5240133/197906397-cf482a5a-1e5d-4e6f-9f35-b494f1db88b8.png)
![image](https://user-images.githubusercontent.com/5240133/197906577-e042d97d-223f-47aa-b936-c50c95a4aa99.png)

adding it to the **tailwind.config.js** classes (from example) **font-title** or **font-body** will become available


As referrer you can check
https://geoffgraham.me/adding-custom-fonts-to-the-wordpress-site-editor-ui/
https://tailwindcss.com/docs/font-family

**Comment by coaststudios** [comment]
I got css images working by placing the images in a public/assets folder in the theme root. then just the image name is needed in `url()`. 
`background-image: url("image.svg");`

see https://vitejs.dev/guide/assets.html#the-public-directory for more info

**Comment by j3hn** [comment]
I got css images working by placing the images in a public/assets folder in the theme root. then just the image name is needed in url().
eg.
`background-image: url("image.svg");`

see https://vitejs.dev/guide/assets.html#the-public-directory for more info

**Comment by szyam** [comment]
I tried a few different things, but wasn't finding a good method. Just recently I tried just letting the 'wp-content' version roll through my build, I received an error that the files were not created and left alone, which is fine since I dont want my images being rebuilt every time, but when I deployed it worked as expected. So, my current css variables look like this:

```
--gray-texture: url('/wp-content/themes/redwood-living/assets/icons/grey-bg-texture.svg');
--wood-texture: url('/wp-content/themes/redwood-living/assets/icons/prefooter-texture.svg');
--top-texture: url('/wp-content/themes/redwood-living/assets/icons/top-texture.svg');
```

This is being rendered in my built css file, now Im going to try to remove the warning from my build process.
